### DISCUSSION TITLE
Send reminder emails from the date of booking

### DISCUSSION THREAD

**Discussion by LicetBovi** [liedekef/events-made-easy#448] [Q&A] [answered]
Hi Franky,

There is an option in the RSVP settings to send reminder emails a few days before the event starts.

Now I have a scenario where it would be desirable to send an email a certain number of days after the booking. Is there an easy way to achieve this?

Thank you very much!

-Klas

**Comment by liedekef** [comment]
See the EME settings, tab RSVP:
![image](https://github.com/liedekef/events-made-easy/assets/75667/55e049d5-10ac-4f87-a367-c58d5f5a3672)

**Reply by LicetBovi** [reply]
Yes, thank you, I am aware of this option. But my question is aimed in a different direction: I would like to send an email a certain number of days after a booking. So the counting of days would have to start with the date of the booking, not the start of the event. This would make quite a difference if the booking is made long before the event starts.

**Reply by liedekef** [reply, marked as answer]
Ah, yes I see. Sorry, I thought the screenshot were the relevant options, but not in your case.
You'll need to hook into eme_events_daily_action, get the list of events, then bookings per event, and then go over each booking creation time. A starter:

```
add_action('eme_events_daily_action','my_custom_eme_events_daily_action_function', 10, 0);
function my_custom_eme_events_daily_action_function() {
        $events = eme_get_events( extra_conditions: 'event_rsvp=1');
        foreach ($events as $event) {
            // do something here, send mails, check attendees, ...
            $bookings = eme_get_bookings_for($event['event_id']);
            foreach ($bookings as $booking) {
                // creation datetime is in $booking['creation_date'], in mysql datetime format
                // call mail: eme_send_mail( ...)
            }
        }
}
```

**Reply by LicetBovi** [reply]
Thank you very much! I will give it a try!
